ui/src/main/webapp/index.html

```diff
@@ -17,48 +17,59 @@
   -->
 
 <html>
-<head>
-    <meta charset=utf-8/>
-    <meta name="viewport"
-          content="user-scalable=no, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0">
-    <title>Gaffer UI</title>
-    <script src="lib/jquery/js/jquery.min.js"></script>
-    <script src="lib/cytoscape/js/cytoscape.min.js"></script>
-    <script src="lib/angular/js/angular.min.js"></script>
-    <script src="lib/angular/js/angular-aria.min.js"></script>
-    <script src="lib/angular/js/angular-animate.min.js"></script>
-    <script src="lib/angular/js/angular-material.min.js"></script>
-    <script src="app/app.js"></script>
-    <script src="app/settings/settings.js"></script>
-    <script src="app/raw/raw.js"></script>
-    <script src="app/table/table.js"></script>
-    <script src="app/graph/graph.js"></script>
-    <script src="app/nav/nav.js"></script>
-    <script src="app/controller.js"></script>
-    <script src="app/graph/buildQuery.js"></script>
-    <link href="lib/angular/css/angular-material.min.css" rel="stylesheet">
-    <link href="lib/bootstrap/css/bootstrap.min.css" rel="stylesheet">
-    <link href="lib/bootstrap/css/bootstrap-theme.min.css" rel="stylesheet">
-    <link href="css/style.css" rel="stylesheet"/>
-</head>
+    <head>
+        <meta charset=utf-8/>
+        <meta name="viewport"
+              content="user-scalable=no, initial-scale=1.0, minimum-scale=1.0, maximum-scale=1.0">
+        <title>Gaffer UI</title>
 
-<body ng-cloak ng-app="app" ng-controller="AppController">
+        <base href="/ui/">
```

Comment:
Can we avoid hardcoding the context-root in here?

Reply:
This has now been removed